ant-node/src/put_validation.rs

```diff
@@ -241,28 +241,44 @@ impl Node {
                 res
             }
             RecordKind::DataOnly(DataTypes::Pointer) => {
-                // Pointers should always be paid for
-                error!("Pointer should not be validated at this point");
-                Err(Error::InvalidPutWithoutPayment(
-                    PrettyPrintRecordKey::from(&record.key).into_owned(),
-                ))
```

Comment:
This would reject any updates meaning pointers can only be set once, making them useless!

Reply:
Not exactly.
`PointData` can still be updated via payment.
i.e. here just block `free updates`

is it design like this ?